data/tlg0007/tlg071/tlg0007.tlg071.perseus-eng3.xml

```diff
@@ -81,10 +81,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
       <body>
       	<div type="translation" xml:lang="eng" n="urn:cts:greekLit:tlg0007.tlg071.perseus-eng3">
          <pb xml:id="v1.p.399"/>
-         <div type="textpart" subtype="section">
-            <head>HOW A MAN MAY BECOME AWARE OF HIS PROGRESS IN VIRTUE (QUOMODO QUIS SUOS IN
-					VIRTUTE SENTIAT PROFECTUS) <lb/>INTRODUCTION</head>
-            <p>The essay on Progress in Virtue is one of Plutarch's polemics against the Stoics,
+            <head>HOW A MAN MAY BECOME AWARE OF HIS PROGRESS IN VIRTUE <foreign xml:lang="lat">(QUOMODO QUIS SUOS IN
+					VIRTUTE SENTIAT PROFECTUS)</foreign></head><lb/> 
+         	<div type="textpart" subtype="section" n="intro">
+         		<p>INTRODUCTION
```

Comment:
This is a good way of dealing with it, but I'd still like the word INTRODUCTION tagged differently. <head> is fine.

Reply:
I followed the tagging instructions you gave Karina on her pull request in my most recent version